src/test/scala/centaur/CallCacheSpec.scala

```diff
@@ -26,6 +28,18 @@ class CallCacheSpec extends FlatSpec with Matchers with ParallelTestExecution {
     }
   }
 
+  "dockerHashNoLookup" should "not lookup docker tag if backend doesn't support docker" in {
+    Workflow.fromPath(DockerHashNoLookup) match {
+      case Valid(w) =>
+        if (w.backends forall CromwellBackendsCompanion.supportedBackends.contains){
+          TestFormulas.runSequentialCachingWorkflows(w, w).run.get
+        } else {
+          succeed
```

Comment:
If I understand you correctly you could potentially build up the tests dynamically like what happens in the standard tests but that seems like a lot of work for little gain here.